### ISSUE TITLE
lab 06_1 오타 수정 문의

### ISSUE THREAD

**Issue by HERIUN** [deeplearningzerotoall/PyTorch#38]
PyTorch also has F.nll_loss() function that computes the negative **loss** likelihood.

= > PyTorch also has F.nll_loss() function that computes the negative **log** likelihood.
